_posts/blog/2021-04-21-jshell-accumulo-feature.md

```diff
@@ -0,0 +1,144 @@
+---
+Title: JShell Accumulo Feature
+
+---
+
+## Overview
+First introduced in Java 9, [JShell][jshell-doc] is an interactive Read-Evaluate-Print-Loop (REPL) 
+Java tool that interprets user's input and outputs the results. This tool provides a convenient way 
+to test out and execute quick tasks with Accumulo in the terminal. This feature is apart of the 
+upcoming Accumulo 2.1 release. If you're a developer and want to get involved in testing, 
+[contact us][contact] or review our [contributing guide][guide]
+
+## Major Features
+* Default JShell script provides initial imports for interacting with Accumulo's API and 
+provided in Accumulo's binary distribution tarball
+
+
+* On startup, JShell Accumulo  will automatically import the `CLASSPATH`, load in a configured 
+environment from user's `conf/accumulo-env.sh`, and invoke `conf/jshell-init.jsh` to 
+allow rapid Accumulo task executions
+
+
+* JShell Accumulo can startup using default/custom JShell script and users can append any JShell 
+command-line [options][jshell-option] to the startup command
+
+## Booting Up JShell Accumulo
+1) Open up a terminal and navigate to Accumulo's installation directory 
+
+2) To startup JShell with **default script** use this command:
+
+```bash
+$ bin/accumulo jshell 
+```
+3) To startup JShell with **custom script** use this command:
+
+```bash
+$ bin/accumulo jshell --startup (file/path/to/custom_script.jsh)
+```
+**Note:** Executing this specific command  _"$jshell"_  will startup JShell, however you will be required to manually import the `CLASSPATH` and the configured environment from `conf/accumulo-env.sh`. In addition, you will need to provide the file directory path to `conf/jshell-init.jsh` before any Accumulo tasks can be performed. Using one of the startup commands above will automate 
+the setup process.
+
+## JShell Accumulo Default Script
+The auto-generated `jshell-init.jsh` is a customizable file located in Accumulo's installation `conf/` directory. Inside `jshell-init.jsh` contains [Accumulo Java APIs][public APIs] formatted as import statements and [AccumuloClient][client] build implementation. On startup the script automatically loads in the APIs and attempts to construct a client. Should additional APIs and/or code implementations be needed, simply append them to `jshell-init.jsh`. Alternatively you can create a separate JShell script and specify the custom script's file path on startup.
```

Comment:
Adding a few commas after some transition words:

```suggestion
The auto-generated `jshell-init.jsh` is a customizable file located in Accumulo's installation `conf/` directory. Inside, `jshell-init.jsh` contains [Accumulo Java APIs][public APIs] formatted as import statements and [AccumuloClient][client] build implementation. On startup, the script automatically loads in the APIs and attempts to construct a client. Should additional APIs and/or code implementations be needed, simply append them to `jshell-init.jsh`. Alternatively, you can create a separate JShell script and specify the custom script's file path on startup.
```

Also, it might be nice to wrap these long lines. I tend to use `vim` to edit files, and generally use `:set tw=65` or then use visual mode to select and `gq` to wrap. I might have to tweak to avoid wrapping specially-formatted markdown, but this works for the most part. It's not strictly necessary to wrap these for Markdown, but it makes looking at the source a little easier sometimes.

Reply:
Sorry about that. I was using the Eclipse IDE when creating this blog post. I'll use vim to word wrap the sentences.

Reply:
It's fine. Use whatever you prefer. I don't know if Eclipse will automatically wrap/format Markdown content or plain text files, but I know it's possible to configure it to show a vertical line at a specific line length, so if you exceed it, you can wrap manually.